### DISCUSSION TITLE
RFP, Permit, and Contract Renewal Language Examples, Feedback, and Showcase

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#137] [General]
We are looking for ongoing feedback and examples around our [draft RFP, permit, and contract renewal language](https://github.com/openmobilityfoundation/governance/blob/main/technical/OMF-CDS-Policy-Language-Guidance.md). And more examples for our public showcase.

Leave your thoughts, and upload or link to your documents in a comment here.

**Comment by mobilitygirl** [comment]
Populus presented on the topic of RFPs, new contracts, and renewals for existing parking technologies to help encourage adoption the Curb Data Specification (CDS) a few months ago, as we were finding very little adoption of the standard amongst vendors that can deliver CDS Events data.

The language that we have shared with cities as guidance for contracts and renewals is available in the following public-facing document:

[CDS City Contracting Guide](https://docs.google.com/document/d/1GYjxmwCcjj0VGBZ7QOtoz0juxz_QHdwe6ET2ncCJgA4/edit#heading=h.hg92oiizdye7)
